docker build fail

apparently the problem is that cmake can't find openssl:

```
$ docker --version
Docker version 26.1.4, build 5650f9b

.../hyperon-experimental$ git branch -v
* main aff2b9f0 Merge pull request #718 from vsbogd/move-out-executable

.../hyperon-experimental$ docker build -t trueagi/hyperon .
[+] Building 173.3s (24/27)                                      docker:default
 => [internal] load build definition from Dockerfile                       0.0s
 => => transferring dockerfile: 1.06kB                
...
...
29.17 -- Conan: Adjusting output directories
29.17 -- Conan: Using cmake targets configuration
29.17 -- Library check found /home/user/.conan/data/libcheck/0.15.2/_/_/package/3538f1114461f03f2ab66edde1e46d4304cd97a0/lib/libcheck.a
29.17 -- Library subunit found /home/user/.conan/data/subunit/1.4.0/_/_/package/1e275394d2b1a4f979603f1ff54ef3a23c31e246/lib/libsubunit.a
29.17 -- Library cppunit_subunit found /home/user/.conan/data/subunit/1.4.0/_/_/package/1e275394d2b1a4f979603f1ff54ef3a23c31e246/lib/libcppunit_subunit.a
29.17 -- Library cppunit found /home/user/.conan/data/cppunit/1.15.1/_/_/package/d8d8aba822aaa76849d2f1bafe4a2a62a9f83b74/lib/libcppunit.a
29.17 -- Conan: Adjusting default RPATHs Conan policies
29.17 -- Conan: Adjusting language standard
29.17 -- Current conanbuildinfo.cmake directory: /home/user/hyperon-experimental/build/hyperonc-prefix/src/hyperonc-build
29.17 -- Conan: Compiler GCC>=5, checking major version 11
29.17 -- Conan: Checking correct version: 11
29.17 CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
29.17   Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
29.17   system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY
29.17   OPENSSL_INCLUDE_DIR)
29.17 Call Stack (most recent call first):
29.17   /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
29.17   /usr/share/cmake-3.22/Modules/FindOpenSSL.cmake:574 (find_package_handle_standard_args)
29.17   CMakeLists.txt:82 (find_package)
29.17 
29.17 
29.17 -- Configuring incomplete, errors occurred!
29.17 See also "/home/user/hyperon-experimental/build/hyperonc-prefix/src/hyperonc-build/CMakeFiles/CMakeOutput.log".
29.18 make[2]: *** [CMakeFiles/hyperonc.dir/build.make:94: hyperonc-prefix/src/hyperonc-stamp/hyperonc-configure] Error 1
29.18 make[1]: *** [CMakeFiles/Makefile2:86: CMakeFiles/hyperonc.dir/all] Error 2
29.18 make: *** [Makefile:101: all] Error 2
------
Dockerfile:35
--------------------
  33 |     WORKDIR ${HOME}/hyperon-experimental/build
  34 |     RUN cmake ..
  35 | >>> RUN make
  36 |     RUN make check
  37 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c make" did not complete successfully: exit code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,7 +3,8 @@ FROM ubuntu:22.04
 RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive \
     TZ=UTC \
-    apt-get install -y sudo git python3 python3-pip curl gcc cmake && \
+    apt-get install -y sudo git python3 python3-pip curl gcc cmake \
+        pkg-config libssl-dev zlib1g-dev && \
     rm -rf /var/lib/apt/lists/*
 
 RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers

--- a/README.md
+++ b/README.md
@@ -82,6 +82,10 @@ Rust (see the Notes at the installation page).
   * GCC (7.5 or later)
   * CMake (3.19 or later)
 
+  To support Git based modules (enabled by default):
+  * OpenSSL library
+  * Zlib library
+
 * Install cbindgen:
 ```
 cargo install --force cbindgen

